### ISSUE TITLE
Hyperparameters for noisyopt

### ISSUE THREAD

**Issue by zohimchandani** [andim/noisyopt#18]
If no hyperparameters are picked, does the algorithm stick to a fixed learning rate or does it apply exponential decay to the learning rate a?

**Comment by andim** [comment]
The learning rate for `minimizeSPSA` is decreased in a non-exponential fashion, as follows: `a/(0.01*niter+k+1)**alpha`
There is currently no capability to use a user-defined learning rate schedule, although that would be easy to implement if of interest.
